### PR TITLE
feat: add power button on dashboard to switch printer on

### LINKED ISSUE #897: Power On button when printer is disconnected.
### Requested feature:

When the printer is off, I need to go into the top right power menu to turn it on, I would love a Power on button in the main view when the printer the Printer switch is detected in the OFF position.
Screen Shot 2022-06-12 at 1 47 22 PM

### Solves the following problem:

Adding a Power On Printer button in the main dashboard area when the Printer switch is Off.

### Additional information:

_No response_

### PR DESCRIPTION
**Pull request by meteyou**
fixes #897

screenshot of the dashboard panel:
image

screenshot from the interface setting:
image

Signed-off-by: Stefan Dej <meteyou@gmail.com>

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -520,6 +520,9 @@
             "FirmwareRestart": "Firmware Restart",
             "KlipperCheck": "Please check if the Klipper service is running.",
             "MoonrakerCannotConnect": "Moonraker can't connect to Klipper!",
+            "PowerOn": "Power on",
+            "PrinterSwitchedOff": "Printer is switched off",
```
Comment (th33xitus) [inline src/locales/en.json:524]:
```suggestion
            "PrinterSwitchedOff": "Printer is powered off",
```

File: src/locales/en.json [hunk 2]
```diff
@@ -1006,6 +1009,8 @@
             "NavigationStyleDescription": "Change navigation appearance",
             "NavigationStyleIconsAndText": "Icons + Text",
             "NavigationStyleIconsOnly": "Icons only",
+            "PowerDeviceName": "Printer power device",
+            "PowerDeviceNameDescription": "Select which Moonraker power device should be used to switch the printer on.",
```
Comment (th33xitus) [inline src/locales/en.json:1013]:
```suggestion
            "PowerDeviceNameDescription": "Select which Moonraker power device should be used to power on the printer.",
```

**Review by th33xitus — APPROVED** [review]

**Comment by LazaroFilm** [comment]
It doesn't seem to work on my printer..
image
image

**Comment by meteyou** [comment]
Did you add the bond_service?

https://docs.mainsail.xyz/overview/quicktips/printer-power-switch

**Pull request merged**